metadata_service/proxy/atlas_proxy.py

```diff
@@ -168,57 +171,93 @@ def _get_user_entity(self, user_id: str) -> EntityUniqueAttribute:
             raise NotFoundException('(User {user_id}) does not exist'
                                     .format(user_id=user_id))
 
-    def _create_reader(self, table_entity: EntityUniqueAttribute, user_guid: str, reader_qn: str) -> None:
+    def _create_bookmark(self, entity: EntityUniqueAttribute, user_guid: str, bookmark_qn: str,
+                         table_uri: str) -> None:
         """
-        Creates a reader entity for a specific user and table uri.
+        Creates a bookmark entity for a specific user and table uri.
         :param user_guid: User's guid
-        :param reader_qn: Reader qualifiedName
+        :param bookmark_qn: Bookmark qualifiedName
```

Comment:
is the concept of Reader still applicable or that should be removed from amundsenatlastypes ?

Reply:
it's still applicable and should be used for frequent-users functionality